docs/rest/ReindexFlowResourceBase.http

```diff
@@ -0,0 +1,170 @@
+# This test flow confirms that reindexing operations can handle search parameters that
+# have base type = "Resource", meaning that they can operate on all types of resources.
+# It first creates two test resources: a patient and an observation.
+# Next, it creates a search parameter resource and triggers a reindexing operation.
+# Finally, it uses the new search parameter to confirm the search parameter
+# is enabled and the resources were successfully reindexed.
+# Note: this assumes that appsettings.json has Security.Enabled = false and Reindex.Enabled = true
+
+@baseUrl = https://localhost:44348
+@contentType = application/json
+
+###
+# Create a patient resource.
+# Copy the patient's ID in the response body to use later.
+# You can paste it here for now: e181cab2-9c1d-4025-916d-d63786ccbdcb
```

Comment:
Could we add variables to store the Patient id, Observation id, Content-Location header?

Reply:
Good idea! Added in d7eb5eb.